config/cmake_ext_mod/HDFTests.c

```diff
@@ -82,12 +82,6 @@ SIMPLE_TEST(timezone=0);
 
 #ifdef PRINTF_LL_WIDTH
```

Comment:
Now that Microsoft supports C99 and HDF5 requires it, this entire test is useless.

Reply:
Done. Hopefully I removed the correct code, am not 100% I got enough/too much.